src/main/java/de/samply/share/broker/rest/Searchbroker.java

```diff
@@ -1271,8 +1274,63 @@ public Response setSiteByName(@HeaderParam(HttpHeaders.AUTHORIZATION) String aut
     }
   }
 
+  /**
+   * Save structured query.
+   * @param queryContainer the structuredQuery and the targets
+   * @return response if the query has been created
+   */
+  @POST
+  @Path("/executeSQ")
+  @Produces(MediaType.TEXT_PLAIN)
+  @Consumes({MediaType.APPLICATION_JSON, MediaType.TEXT_PLAIN})
+  @APIResponses({
+      @APIResponse(
+          responseCode = "200",
+          description = "ok",
+          content = @Content(
+              mediaType = MediaType.APPLICATION_JSON,
+              schema = @Schema(implementation = String.class))),
+      @APIResponse(responseCode = "500", description = "Internal Server Error")
+  })
+  @Operation(summary = "Save query in searchbroker database")
+  public Response executeSQ(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
+      @Parameter(
+          name = "structuredQuery",
+          description = "Query as a JSON object",
+          schema = @Schema(implementation = QueryContainer.class))
+          QueryContainer<String> queryContainer) {
+    try {
+      if (auth == null || !checkBasicAuth(auth, "abide.username", "abide.password")) {
```

Comment:
We don't like to have project names in the code. Accepting a structured query is a feature that it independent of a concrete project like abide. Why do we have a separate account for executing structured queries?